akka-actor/src/main/scala/akka/actor/ActorSelection.scala

```diff
@@ -194,7 +195,7 @@ object ActorSelection {
         if ((x.indexOf('?') != -1) || (x.indexOf('*') != -1)) SelectChildPattern(x)
         else if (x == "..") SelectParent
         else SelectChildName(x)
-    })(scala.collection.breakOut)
+    }).to(immutable.IndexedSeq)
```

Comment:
wasn't the trick to use `elements.iterator` to avoid creating the "heavy" intermediate collection?

Reply:
This is the trick:
https://github.com/scala/scala-collection-compat/pull/70/files#diff-3a68dcebdb0ce58a1563a05d1eedcf55R10

Reply:
yes. if it were 2.13-only code you could use `.view`, but `.iterator` is the cross-buildable solution